pkgs/applications/graphics/dungeondraft/default.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv
+, lib
+, requireFile
+, dpkg
+, xorg
+, libGL
+, udev
+, pulseaudio
+, libkrb5
+, zlib
+}:
```

Comment:
Arguments should be listed in the order they are used, with the exception of lib, which always goes first. (see https://nixos.org/manual/nixpkgs/stable/#sec-syntax last point).

Reply:
that has recently been changed

  * #247424